lib/utils.sh

```diff
@@ -1,3 +1,4 @@
+#! /usr/bin/env bash
```

Comment:
This shouldn't be needed since this file is `source`d by the actual callback scripts.

Reply:
Removed